src/Workspaces/Core/Portable/FindSymbols/IStreamingFindReferencesProgress.cs

```diff
@@ -32,10 +32,6 @@ public SymbolGroup(ImmutableArray<ISymbol> symbols)
         {
             Contract.ThrowIfTrue(symbols.IsDefaultOrEmpty);
 
-            // We should only get an actual group of symbols if these were from source.
-            // Metadata symbols never form a group.
-            Contract.ThrowIfTrue(symbols.Length >= 2 && symbols.Any(s => s.Locations.Any(loc => loc.IsInMetadata)));
```

Comment:
this assert just wasn't sensical in teh case of symbols that span into metadata (like namespaces).  so i'm just removing it entirely.  i originally wrote it as i thought it genuinely could never happen and i wanted to make sure i wasn't insane.  turns out i am insnae and this is normal.